testrunner

```diff
@@ -255,6 +255,17 @@ install_packages() {
         mgr=dnf
     fi
 
+    # This is hacky and sad. Preemptively try to refresh
+    # yum/dnf cache to work around flaky distro infras.
```

Comment:
rpm-md metadata + fedora's model of deleting packages is fundamentally racy.  The only fix is to have a local repo that has multiple copies of both metadata and packages.